qemu/tests/cfg/virtio_console.cfg

```diff
@@ -227,11 +231,14 @@
                         - boot_too_much_ports:
                             # arm doesn't use pci bus
                             no aarch64
-                            extra_params = " -device virtio-serial-pci,max_ports=32"
+                            extra_params_rhel = " -device virtio-serial-pci,max_ports=32"
+                            extra_params = " -device virtio-serial-pci,max_ports=512"
                             start_vm = no
                             virtio_console_test = failed_boot
-                            virtio_console_params = "maximum ports supported: 31"
+                            virtio_console_params_rhel = "maximum ports supported: 31"
+                            virtio_console_params = "maximum ports supported: 511"
                             virtio_ports = ""
+                            version_params = "1\.\d\.\d"
```

Comment:
I'd suggest change `version_params` to `qemu_verison_patten`.
I am not sure if use version number is the best way. 
I saw you use `qemu_binary = utils_misc.get_qemu_binary(params)` below,
based on what I know rhev6 is qemu－kvm－rehv 0.x.x

Are you trying to make it compatible for rhev7 and rehl7 ?

Reply:
Yes, I am trying to make it compatible for rhev7 and rhel7, and rhel6 if possible, since the "max_ports" now is only extended on rhev7 version, I think if we match the version pattern with rhev7, the lower pattern will be well covered.